exercises/bowling/example.rb

```diff
@@ -20,7 +20,7 @@ def roll(pins)
   end
 
   def validate(pins)
-    raise 'Invalid number of pins' unless (RULES[:MIN]..RULES[:MAX]).cover?(pins)
+    raise 'Invalid number of pins' unless (PINS[:MIN]..PINS[:MAX]).cover?(pins)
```

Comment:
Nice catch.  Do we test for this?  I don't think it is an important aspect in relation to what things are learned from the exercise.  But definitely fodder for discussion in solutions!

I think when we were putting this together we didn't even consider it.